src/toolchain/toolchain.ts

```diff
@@ -488,11 +488,12 @@ export class SwiftToolchain {
      * Library path for swift-testing executables
      */
     public swiftTestingLibraryPath(): string | undefined {
+        let result = "";
         const base = this.basePlatformDeveloperPath();
-        if (!base) {
-            return undefined;
+        if (base) {
+            result = `${path.join(base, "usr/lib")}:`;
         }
-        return path.join(base, "usr/lib");
+        return `${result}${path.join(this.toolchainPath, "lib/swift/macosx/testing")}`;
```

Comment:
I think it's just missing `usr/` prefix, now
```suggestion
        return `${result}${path.join(this.toolchainPath, "usr/lib/swift/macosx/testing")}`;
```

Reply:
`this.toolchainPath` already has `/usr` prefixed in the path, the comment on it says "one folder up from the swift bin path": https://github.com/swiftlang/vscode-swift/pull/1521/files#diff-219ebe526be52811261023900cb320d9e51723308f63891304bfe53dcab033f1R107

[The commit message has a bit of history](https://github.com/swiftlang/vscode-swift/pull/606) there that predates my time, it may not be true anymore, however it would be a bigger change to update the toolchainPath everywhere.

Reply:
Ah, ok if it already has that prefix then great! 👍